### PR TITLE
Update installation guide for the Jira plugin

### PR DESCRIPTION
**Pull request by mcalus3**

### CODE REVIEW

File: content/plugins/descriptions/jira.yaml [hunk 1]
```diff
@@ -79,16 +79,16 @@ gettingStarted:
...
-          jira/project-key: [example-jira-project-key]
-          jira/component: [example-component] // optional, you might skip this value to fetch data for all components
+          jira/project-key: &lt;example-jira-project-key>
+          jira/component: &lt;example-component> # optional, you might skip this value to fetch data for all components
   - intro: Get and provide `JIRA_TOKEN` as env variable in following format
     language: yaml
     code: |
       "Basic base64(jira-mail@example.com:JIRA_TOKEN)"
-      e.g. `Basic: ZXhhbXBsZV9qaXJhQGV4YW1wbGUuY29tOjU1Q3NUSEoxWW1oTVdJSFptdGJXNUUxOA==`
+      e.g. `Basic ZXhhbXBsZV9qaXJhQGV4YW1wbGUuY29tOjU1Q3NUSEoxWW1oTVdJSFptdGJXNUUxOA==`
```
Comment (dtuite) [inline content/plugins/descriptions/jira.yaml:91]: I am not sure that I understand what I need to do in order to use this plugin after reading this. 

Can we simplify it or perhaps use the notes section of this page to show the user where to get this token and how to pass it to Backstage in an environment variable?

**Review by dtuite — DISMISSED** [review]

File: content/plugins/descriptions/jira.yaml (hunk 1 above)
Comment (mcalus3) [reply content/plugins/descriptions/jira.yaml:91]: I've got the proper text but don't know if I can format it properly in this template:
image

**Comment by mcalus3** [comment]
Opdated with a note, can be merged:
image

**Comment by martina-if** [comment]
LGTM but David should also take a look

File: content/plugins/descriptions/jira.yaml (hunk 1 above)
Comment (dtuite) [reply content/plugins/descriptions/jira.yaml]: @mcalus3 The note is great and it is much easier to understand the instructions now. Two more small things are needed:

1. The installation steps should tell the user that they need to provide a `JIRA_TOKEN` environment variable and they should check the instructions at the bottom of the page to learn how to generate that. It's too easy to miss the note under the screenshot.
2. The note should show the user how to pass the environment variable to Backstage. i.e. `env JIRA_TOKEN=xyz yarn dev`

This is ready to go once those two small things are in place.

Comment (mcalus3) [reply content/plugins/descriptions/jira.yaml]: Done

**Review by dtuite — APPROVED** [review]

**Pull request merged**